src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+ul,
+li,
+body,
+nav {
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding-left: 50px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.nav {
+  display: flex;
+  justify-content: end;
+  margin-right: 50px;
+  margin-left: auto;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+a.nav__link {
+  display: flex;
+  text-decoration: none;
+  color: black;
+  padding: 22.5px 0;
+  margin: 0 10px;
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+.nav__item:first-child a {
+  margin-left: 0;
+}
+
+.nav__item:last-child a {
+  margin-right: 0;
+}
+
+a.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+a.is-active::after {
+  content: "";
+  background-color: #00acdc;
+  height: 0;
+  border-width: 2px;
+  border-radius: 4px;
+  border-style: solid;
+  display: flex;
+  margin-top: 33px;
```

Comment:
mb just `bottom: 0` suits better?
if we change height of the element we will need to go here and change margin top too(